docker-compose.yml

```diff
@@ -84,8 +82,8 @@ services:
         CLI_IMAGE: *cli-image # Inject the name of the cli image
     depends_on:
       - php
-    << : *default-volumes # loads the defined volumes from the top
-    << : *default-user # uses the defined user from top
+    # loads the defined volumes and user from the top
+    <<: [*default-volumes, *default-user]
```

Comment:
I am unsure as to what these changes do, but if you're confident in them, I trust your judgement,

Reply:
Docker compose was failing in CI because the syntax has changed regarding merging defaults